Source/Core/Utilities/Tooling/BaseCommandLineOptions.cs

```diff
@@ -55,60 +50,56 @@ public Configuration Parse()
             return Configuration;
         }
 
-        #endregion
-
-        #region protected methods
-
         /// <summary>
         /// Parses the given option.
         /// </summary>
         /// <param name="option">Option</param>
         protected virtual void ParseOption(string option)
         {
-            if (option.ToLower().Equals("/?"))
+            if (this.IsMatch(option, @"[\/|-]?$"))
             {
                 this.ShowHelp();
                 Environment.Exit(0);
             }
-            else if (option.ToLower().StartsWith("/s:") && option.Length > 3)
+            else if (this.IsMatch(option, @"[\/|-]s:") && option.Length > 3)
```

Comment:
This should rather be: `this.IsMatch(option, @"^[\/|-]s:")` otherwise, even `ab/s:blah` will also match.

Reply:
Nice catch!